src/pages/PdfViewerPage/PdfViewerPage.tsx

```diff
@@ -176,7 +181,10 @@ const PdfViewerPage: FC<PdfViewerPageProps> = ({
           {referenceDetailsMountNode &&
             createPortal(<Reference />, referenceDetailsMountNode)}
           {knowledgeGraphMountNode &&
-            createPortal(<TestTool />, knowledgeGraphMountNode)}
+            createPortal(
+              <Button onClick={sendToKG}>Go to KG</Button>,
```

Comment:
Instead of adding button here, call <KnowledgeGraph> component here where <TestTool> was

Reply:
I don't know exactly what you mean here, could you expand a bit, please?

Reply:
Can you get on a call?

Reply:
Yea, sure.